requirements-ci.txt

```diff
@@ -13,3 +13,4 @@ matplotlib
 requests
 rich
 tomli # Drop once minimum Python version is 3.11
+tach
```

Comment:
We should really standardize this, but we don't need to put dependencies in both `requirements-dev.txt` and `requirements-ci.txt`. The test workflows use both files to install dependencies, and it gives us multiple sources of truth, which can lead to issues in the future if we pin versions.

I'd remove it from here and keep it in `requirements-dev.txt` since it's needed for the pre-commit hook.
```suggestion
```

Reply:
These will all likely be blown away when we move to `uv` anyway.